src/main/java/org/kiwiproject/champagne/jdbi/UserDao.java

```diff
@@ -37,9 +37,9 @@ public interface UserDao {
     @SqlQuery("select count(*) from users")
     long countUsersIncludingDeleted();
 
-    @SqlUpdate("update users set deleted = true where id = :id")
+    @SqlUpdate("update users set deleted = true, updated_at = now() where id = :id")
```

Comment:
In the other DAO and here, we're using `now()`. But in the migration, we're using `current_timestamp`.  Since `current_timestamp` is a SQL standard, perhaps we should prefer that. See https://dba.stackexchange.com/questions/63548/difference-between-now-and-current-timestamp and the referenced section of the Postgres  manual: https://www.postgresql.org/docs/current/functions-datetime.html#FUNCTIONS-DATETIME-CURRENT